geopandas/vectorized.pyx

```diff
@@ -1266,6 +1266,11 @@ class GeometryArray(object):
     def envelope(self):
         return geo_unary_op('envelope', self.data)
 
+    def exterior(self):
+        out = geo_unary_op('exterior', self.data)
+        out.parent = self
```

Comment:
Do you know why this is needed? (or just noticed that it otherwise gives segfaults / memory errors?) 
if so, can you add a comment ?